start/sw/web-app/laravel5.md

```diff
@@ -9,41 +9,51 @@ order: 4
 
 Laravel is a high-level PHP Web framework.
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
+
 
 #### Install MariaDB Relational Database Server
 Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
 
-#### Install PHP
-Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
+Or you can install by : 
+```bash
+# install MariaDB
+$ dnf install mariadb-server mariadb
+# start MariaDB
+$ systemctl enable --now mysql
+# initialize MariaDB
+$ sudo mysql_secure_installation
+```
 
-#### Install PHP Extensions
+#### Install PHP and all its dependencies
+Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
 ```bash
-# Laravel required PHP extensions.
-$ sudo dnf install php php-common php-cli php-pdo php-mbstring php-zip php-xml
-# Restart Apache
-$ sudo systemctl restart httpd
+# It will install all required extensions, perk of Fedora
+$ sudo dnf install composer
 ```
 
-#### Install Composer
-We will need composer to install laravel and it's dependencies
+#### Install Valet Linux
+We will need to set it up for easier development. Valet is original designed for MacOS by the Laravel Team; Valet Linux is a port to that. It setups up the neccesary environments for PHP Development, including a test domain. Please do not run valet as root. 
+For More Information, you can check them out here: 
+https://cpriego.github.io/valet-linux/ 
+https://laravel.com/docs/5.7/valet
 
 ```bash
-$ curl -sS https://getcomposer.org/installer | php
-$ sudo mv composer.phar /usr/local/bin/composer
-$ chmod +x /usr/local/bin/composer
-$ composer -V
+# Place the ~/.config/composer/vendor/bin directory in your PATH so the composer 
+# global executable can be located by your system.
+$ echo 'export PATH="$PATH:$HOME/.config/composer/vendor/bin"' >> ~/.bashrc
+# Install valet 
+$ composer global require cpriego/valet-linux
+# Initialize valet
+$ valet install
 ```
 
 #### Install laravel installer
 
 ```bash
 $ composer global require "laravel/installer"
```

Comment:
Ok, so now we're getting to work with laravel at last. The all previous were dependencies. Could you maybe denote that somehow in the beginning like:
```
## Dependencies for laravel on Fedora
 - mysql
 - composer
 - valet linux
```
Anyways, aren't those tools worthy enough to have their own standalone page? Like f.e.:
 - http://developer.fedorainfracloud.org/tools/vagrant/about.html
 - http://developer.fedorainfracloud.org/tools/docker/about.html

Reply:
vagrant and docker are vm-like technologies, and it is out of scope for setting up Laravel on Local Environments ...

Reply:
Yes it is. I didn't mean to include the content in Laravel page, on the contrary, I was thinking of moving the content elsewhere and pointing to it from here. Anyway it was just a suggestion and the move can be done in later iterations.

My main point was, though, that there was no word on _Why_ you're installing those packages/tools. It would be nice to include the information, like stated above, in some `dependencies list`, or similar. As the developer can have already those installed, they can skip the step, etc.. (The list would be hyper-linked.)